packages/wdio-sauce-service/src/launcher.js

```diff
@@ -15,7 +15,7 @@ export default class SauceLauncher {
         }, config.sauceConnectOpts)
 
         config.protocol = 'http'
-        config.host = 'localhost'
```

Comment:
@christian-bromann pay attention. 
I don't know very well how `wdio-sauce-service` and why we need it here